src/blocks/battery_block.rs

```diff
@@ -44,15 +43,20 @@ impl Block for BatteryBlock {
                                 percent_charged = percent;
                             }
                         }
+                        let current = std::fs::read_to_string(supply.path().join("current_now"))
+                        .ok()
+                        .and_then(|v| v.trim().parse::<f64>().ok());
+                        
+                        let voltage = std::fs::read_to_string(supply.path().join("voltage_now"))
+                        .ok()
+                        .and_then(|v| v.trim().parse::<f64>().ok());
+                        
+                        let watts = if let (Some(current), Some(voltage)) = (current, voltage) {
+                            (current * voltage) / 1_000_000_000_000.0
+                        } else {
+                            0.0
+                        };
```

Comment:
Don't we want the old logic as a fallback?

Reply:
it seems power_now is not a standard feature and only present with lenovo-laptops (it comes from tp_smapi and does pretty much the same: https://github.com/linux-thinkpad/tp_smapi/blob/bee13fd060cc0e8fed631bfd9e98f6cac715ce98/tp_smapi.c#L1030 ->  https://github.com/linux-thinkpad/tp_smapi/blob/master/tp_smapi.c#L687)